### DISCUSSION TITLE
Welcome to RealCamera Discussions! - English and 中文 recommended

### DISCUSSION THREAD

**Discussion by xTracr** [xTracr/RealCamera#39] [Announcements]
#### Languages: English and 中文
We’re using Discussions as a place to discuss improvements and issues of the experimental feature. 
We hope that you:
 - Ask questions you’re wondering about.
 - Share ideas.
 - Engage with other community members.
 - Welcome others and are open-minded. 

我们将 "讨论区 "作为讨论实验功能的改进和问题的场所。
我们希望您能
 - 提出您想知道的问题。
 - 分享想法。
 - 与其他社区成员互动。
 - 欢迎他人并持开放态度。

**Comment by lucia2048** [comment]
能否跟‘是，史蒂夫模型’进行兼容？我可以帮忙联系‘是，史蒂夫模型’的模组开发作者。
Is it compatible with the 'Yes Steve model'? I can help contact the author of the module development of 'Yes, Steve Model'.

**Reply by xTracr** [reply]
0.6.0-alpha.1版本测试过是能的，虽然还有些小问题，并且禁用模型渲染的部分还没做

**Reply by lucia2048** [reply]
好的，谢谢，这个版本可以自定义摄像机绑定在哪个骨骼上了吗

**Reply by xTracr** [reply]
可以绑在顶点/平面上，所以理论上兼容性很强，但设置起来稍微有点麻烦

**Reply by lucia2048** [reply]
设置起来有点麻烦，向上矢量的设置有些麻烦，因为部分模型的头顶并不是平面，同时设置好一个模型之后，如果直接切换模型会导致游戏崩溃

**Reply by lucia2048** [reply]
如果能把摄像机绑定在模型的组上可能会好点，目前的设置因为需要选择块，导致不同的模型无法通用设置，如果设置一个通用的组名，作为摄像机的绑定点会好很多

**Reply by xTracr** [reply]
我当时测试过换模型，不过是偶尔会崩溃，等我过几天放假了我会去处理的。
通用设置的话，目前没有什么好的想法，因为基本上各个模组都有自己的一套模型。
等我把禁用模型渲染的部分做了，我还要再加一点筛选条件什么的，比如只显示与当前选中的面有公共顶点或者垂直的面什么的，到时候应该会更方便。

**Comment by IndigoWing** [comment]
可以添加一下对mod （龙之生存）的支持吗

**Reply by xTracr** [reply]
我测试了一下，大体上是支持的，但是目前配置起来有点麻烦

**Comment by SCP3065** [comment]
Got a quick question if you know english. Do you know this mod called Do A Barrel Roll? If you do, will there be any conflicts with Real Camera and Do a Barrel Roll? Just need to know before putting it into Minecraft

**Reply by SCP3065** [reply]
Nvm I saw the configuration slide and it is compatible with Do a Barrel Roll. Sorry

**Comment by Cyrenix** [comment]
will the new feature be able to suport Customizable player models?

**Reply by xTracr** [reply]
I guess it supports, but I don't have a model of cpm, so I haven't tested before

**Comment by Cyrenix** [comment]
will this get Gravity Changer support in the future? first person model view gets a bit weird when not in normal gravity
